Add page for authors content

fzipi opened the pull request:
In this PR I changed the authors of a blog post to be a taxonomy.

That way pages are generated easily with the respective content.

DinisCruz commented:
@RandomFractals can you review this and solve the conflict (and make sure it still works ok (the new author feature)

fzipi commented:
@DinisCruz I updated this one to the last master, and changed post content author to be a list of "authors".

This is what you have now:
![screenshot from 2018-02-24 00-45-57](https://user-images.githubusercontent.com/3012076/36625328-499cf6a6-18fc-11e8-9383-dc72d5b09e6a.png)
The author of each post is a link, and when followed it will lead you to the list of the author's content:
![screenshot from 2018-02-24 00-45-44](https://user-images.githubusercontent.com/3012076/36625334-60e23178-18fc-11e8-9055-bda8a0f99ac8.png)
